docroot/themes/custom/drupal4gov/sass/base/forms.scss

```diff
@@ -297,3 +297,33 @@ select {
     font-weight: bold;
   }
 }
+
+.form-item-full-name- {
```

Comment:
@WaseemZ this can be consolidated to:

.form-item-full-name-,
.form-item-last-name-,
.form-item-email,
.form-item-subject,
.form-item-body  {
  label {
    font-weight: bold;
  }
}

Reply:
Done.